app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -114,6 +116,19 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  add_properties(:HighlightUpdate) do
+    property :color do
+      key :type, :string
+      # remove the anchors because swagger-codegen always escapes them
+      key :pattern, ::Highlight::VALID_COLOR.inspect[1..-2]
```

Comment:
maybe we should store this "concept" on the model, so we can refer to one place for it.

Reply:
Can you elaborate?

Reply:
e.g, as `::Highlight::VALID_COLOR_WOUT_ANCHORS`